backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/service/HotelService.java

```diff
@@ -0,0 +1,40 @@
+package com.chinguv45tier3team53.tripplannerrest.service;
+
+import com.chinguv45tier3team53.tripplannerrest.dao.HotelRepository;
+import com.chinguv45tier3team53.tripplannerrest.entity.Hotel;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class HotelService {
+    private final HotelRepository repository;
+
+    public List<Hotel> findAll() {
+        return repository.findAll();
+    }
+
+    public Hotel findById(long id) {
```

Comment:
i think it is cleaner to have something like
`Hotel hotel = repository.findById(id).orElseThrow()`
and also throwing something more appropriate like entity not found exception

Reply:
you are right, I'll change it